### PR TITLE
Add Noah as a Slack moderator.

### PR DESCRIPTION
**Pull request by coderanger**
Whoops, forgot to add myself in here before.

### CODE REVIEW

**Comment by nikhita** [comment]
@coderanger I hate to be a stickler, but can you update the commit message to `Add coderanger as slack moderator` or something like that? It turns out to be helpful while doing a `git blame` sometimes :P

**Comment by coderanger** [comment]
@nikhita History rewritten, you never saw anything ::mind trick::

**Comment by castrojo** [comment]
/lgtm

Though Ihor's probably going to ask you to fix the timezone to match everyone else. :D

File: communication/moderators.md [hunk 1]
```diff
@@ -136,6 +136,7 @@ Moderators seats: 9
 - Paris Pittman (@parispittman) - PT
 - Jorge Alarcon (@alejandrox1) - ET
 - Yang Li (@idealhack) - CST
+- Noah Kantrowitz (@coderanger) - America/Los_Angeles
```
Comment (nikhita) [inline communication/moderators.md]: Can we all use UTC? :stuck_out_tongue:

Comment (nikhita) [reply communication/moderators.md]: Oh wait...UTC doesn't observe DST.

I'll leave this to our timezone experts. :woman_shrugging:

Comment (idvoretskyi) [reply communication/moderators.md]: @coderanger Can you please fix the timezone to match everyone else? :D

PS. I gonna make the TZ's more readable with https://github.com/kubernetes/community/issues/3380, but for now please do it in an abbreviation format.

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Comment by natetech333** [comment]
image

**Pull request merged**